plugins/statuslines/lualine.nix

```diff
@@ -36,7 +36,7 @@ with lib; let
               Enables the display of icons alongside the component.
             '';
 
-            icon = helpers.mkNullOrOption types.str ''
+            icon = helpers.mkNullOrOption (types.either types.str helpers.rawType) ''
```

Comment:
```suggestion
            icon = helpers.mkNullOrOption (with types; either str (attrsOf anything)) ''
```

Reply:
Thanks for showing me the change.
I am trying to test locally before pushing the change.
I am having a problem on the generation. 

I think something else needs to be define maybe here https://github.com/nix-community/nixvim/blob/main/plugins/statuslines/lualine.nix#L214. in order to handle the unkeyed value for the icon. 
I've tried several things with no success.  maybe you can point me to some example to use as base. 
Thanks.